### ISSUE TITLE
Sort for time in milliseconds doesn't work 

### ISSUE THREAD

**Issue by xaksis** [xaksis/vue-good-table#50]

**Comment by xaksis** [comment]
fixed in v 1.10.1

**Comment by ssuess** [comment]
I just upgraded to this, and sorting on my unixtime date column (after formatting the date) does not work. Here is my formatting code for the date column:

```
{ label: 'Date',
          field: function (row) {
            var X = row.date * 1000
            return X
          },
          width: '15%',
          sort: true,
          type: 'date',
          inputFormat: 'x',
          outputFormat: 'MMM D, YYYY '
        }
```
